extensions/_beacons/http_status.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+'''
+Beacon to manage and report the status of a server status endpoint.
+Fire an event when specified values don't match returned response.
+
+'''
+
+# Import Python libs
+from __future__ import absolute_import, unicode_literals
+import logging
+import operator
+import re
+import requests
+import salt.utils
+from salt.ext.six.moves import map
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'http_status'
+
+comparisons = {
+  '=': operator.eq,
```

Comment:
I would argue that this should be `==` to be consistent with standard program syntax, but :man_shrugging: